docs/guides/websites/ecommerce/install-magento-2-4-on-centos-8/index.md

```diff
@@ -0,0 +1,649 @@
+        AllowOverride All
+    </Directory>
+
+</VirtualHost>
+
+{{< /file >}}
+5.  Add an `IncludeOptional` directive to `etc/httpd/conf/httpd.conf` to parse this directory:
+
+    {{< file "etc/httpd/conf/httpd.conf" aconf >}}
+    IncludeOptional conf.d/*.conf
+{{< /file >}}
+6.  SELinux might prevent Apache from writing to these logs or even from starting up with these settings. If this occurs, navigate to the `var/www/html/example.com` directory and run the following commands:
+
+        semanage fcontext -a -t httpd_sys_rw_content_t 'logs'
+        sudo semanage fcontext -a -t httpd_sys_rw_content_t 'logs'
+7.  If Apache is not already configured to activate at boot, add an entry to `systemctl` to this effect. Then restart the web server:
+
+        sudo systemctl start httpd.service
+        sudo systemctl enable httpd.service
+        sudo systemctl restart httpd.service
+
+{{< note >}}
+If you require more information on setting up Apache or configuring virtual hosts, consult Linode's [Apache on CentOS 8](/docs/web-servers/apache/how-to-install-apache-web-server-centos-8) guide.
+{{< /note >}}
+
+{{< note >}}
+nginx 1.x can also be used as the Magento web server.
+{{< /note >}}
+
+### SQL Database
+
+MySQL 8.0 is recommended, although MariaDB version 10.4+ can also be used. All Magento SQL statements are compatible with MariaDB. Execute the following steps to prepare your database for the Magento installation.
+
+1.  Ensure MySQL is installed. Confirm its release version and upgrade it if necessary.
+
+        sudo dnf install mysql-server
+        mysql -V
+2.  Ensure the MySQL server is running. If not, start your server and verify its status with the following commands:
+
+        sudo systemctl start mysqld.service
+        sudo systemctl status mysqld
+3.  Run the `mysql_secure_installation` utility to ensure your database is more secure. This utility walks you through your security settings and allows you to set a root password.
+
+        mysql_secure_installation
+4.  Log in to mysql as root:
+
+        mysql -u root -p
+5.  Create a database and user account for Magento. Replace `dbpassword `with your own strong password (the exclamation point "!" must not be used in an MySQL password). You can also change the user name to something more specific (in this case, ensure you make the same substitution in the GRANT command).
+
+        CREATE database magento;
+        CREATE user 'magento' IDENTIFIED BY 'dbpassword';
+        GRANT ALL ON magento.* TO 'magento';
+        FLUSH PRIVILEGES;
+        EXIT;
+6.  Log in to the database as the new user and verify it is accessible:
+
+        mysql -u magento -p
+7.  Record the name of the database and password you created and store this information in a safe place for later.
+8.  If mysql is not already configured to become active at start-up time, add this entry to `systemctl`:
+
+        systemctl enable mysqld
+
+### PHP
+
+Magento recommends the use of PHP 7.4.0 or higher.
```

Comment:
This is a requirement not a recommendation:

Magento supports PHP 7.3.0 or later. Please read https://devdocs.magento.com/guides/v2.4/install-gde/system-requirements-tech.html